Services/CO.CDP.OrganisationInformation.Persistence/Migrations/20241030163144_AddExpiresOnToPersonInvite.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CO.CDP.OrganisationInformation.Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddExpiresOnToPersonInvite : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTimeOffset>(
+                name: "expires_on",
+                table: "person_invites",
+                type: "timestamp with time zone",
+                nullable: true);
```

Comment:
what will happen to any person invites that already exist in the service? They won't have expiration dates - will they flow ok through the rest of your code?

Not necessarily a request for a change - just something to think about. If it already works fine, then that's great 👍🏻

Reply:
Or, do we need another migration to calculate expiration dates for the invites that are already in the db?

Reply:
Thats a really good point and I tested it with legacy data on my local. What will happen is any person invites without an expiredOn will be displayed as normal. When the user creates a new invite for the email in that organisation it will look for any other matching person invites that have not yet been expired and expire them. 

So, at the moment there may be legacy data where someone has created multiple person invites for an email / org ... As soon as they request yet another person invite it will expire all the others, fixing the current duplicate problem and display only the one just created. 

Let me know if you have any further questions.